packages/stacks-docs/product/components/tags.html

```diff
@@ -37,15 +43,15 @@
 {% highlight html %}
 <div class="d-flex g4">
     <a class="s-tag" href="#">jquery</a>
-    <span class="s-tag">javascript <button class="s-tag--dismiss">@Svg.ClearSm</button></span>
-    <a class="s-tag" href="#"><img class="s-tag--sponsor" src="https://i.stack.imgur.com/tKsDb.png" width="16" height="18" alt="Google Android"> android</a>
+    <span class="s-tag" tabindex="0">javascript <button class="s-tag--dismiss"><span class="v-visible-sr">Dismiss</span>@Svg.ClearSm</button></span>
```

Comment:
I'm not sure adding `tabindex="0"` is the best move here. I recommend reserving tabbing for interactive elements. Generally, these dismissable tags will all be grouped together so I think context mixed with my suggestion for slightly more descriptive screen reader text on the dismiss button is preferable. Let me know if I'm overlooking some details.

```suggestion
    <span class="s-tag">javascript <button class="s-tag--dismiss"><span class="v-visible-sr">Dismiss</span>@Svg.ClearSm</button></span>
```

Reply:
Thanks for the feedback Dan. I forgot that these tags aren't interactable, only the dismiss button is. My intention behind adding the tabindex was so I could get my screen reader to read the tag itself. Without the tabindex, when using keyboard navigation, I can only focus the dismiss button and the screen reader just reads "dismiss button"; no way to get which tab I'm dismissing.

But your idea to include the tag's name in the dismiss button's screen reader content solves that concern for me so I removed the tabindex's now 👍🏼 https://github.com/StackExchange/Stacks/pull/2045/commits/89cdf247a210fd870ae7b03424635e91a90f53c3

It does cause a little inconsistency in our Svelte component as [we are not automatically injecting the tag name](https://github.com/StackExchange/Stacks/blob/8e628a58bd6b9d96a0c79637d4540af21114632e/packages/stacks-svelte/src/components/Tag/Tag.svelte#L196) into the screen reader content there 🤷🏼

Reply:
> It does cause a little inconsistency in our Svelte component as [we are not automatically injecting the tag name](https://github.com/StackExchange/Stacks/blob/8e628a58bd6b9d96a0c79637d4540af21114632e/packages/stacks-svelte/src/components/Tag/Tag.svelte#L196) into the screen reader content there 🤷🏼

Yeah, if we were only allowing text to be passed in place of the `children` snippet, we could construct default text for the dismiss button that includes the tag name (like `Dismiss ${tagName} tag`) but I think we'll need to rely on consumers setting the appropriate value for now. I'm ok with that but I agree it's not ideal.